src/test/java/core/basesyntax/service/operations/ReturnOperationTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.exceptions.InvalidFruitException;
+import core.basesyntax.service.exceptions.InvalidOperationException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnOperationTest {
+    private FruitTransaction transaction;
+
+    @BeforeEach
+    void beforeAll() {
+        Storage.clear();
+        Storage.put(new Fruit("apple"), 10);
+    }
+
+    @Test
+    void apply_correctOperation_Ok() {
+        transaction = new FruitTransaction(
+                FruitTransaction.Operation.RETURN,new Fruit("apple"),1);
+        OperationHandler operation = new ReturnOperation();
+        operation.apply(transaction);
+        Map<Fruit, Integer> expected = new HashMap<>();
+        expected.put(new Fruit("apple"),11);
+        Assertions.assertEquals(expected,Storage.copy());

```

Comment:
You are directly comparing HashMap objects using 'assertEquals'. It's recommended to compare the specific values within the map for a more precise comparison.

Reply:
ok,done